tests/driver.sh

```diff
@@ -296,4 +296,17 @@ int main()
 }
 EOF
 
+# break from while
+try_ 1 << EOF
```

Comment:
The following test program might be redundant if the test cases in `loop` section are enabled.